cmd/history_index.cpp

```diff
@@ -165,9 +164,13 @@ int main(int argc, char* argv[]) {
             * cannot guarantee keys are sorted amongst different calls
             * of this stage
             */
+            auto target_table{txn->open(index_config, MDB_CREATE)};
             size_t target_table_rcount{0};
             lmdb::err_handler(target_table->get_rcount(&target_table_rcount));
             unsigned int db_flags{target_table_rcount ? 0u : (uint32_t)MDB_APPEND};
+            if (db_flags == MDB_APPEND) {
+                txn->open(index_config)->clear();
+            }
```

Comment:
What is the point of this ?
When db_flags == MDB_APPEND it means target_table is already empty.
Why re-empty it ?